internal/jsre/deps/web3.js

```diff
@@ -3143,7 +3143,7 @@ module.exports = {
         return new Error(message);
     },
     ConnectionTimeout: function (ms){
-        return new Error('CONNECTION TIMEOUT: timeout of ' + ms + ' ms achived');
```

Comment:
It's the external web3.js library used in the go-ethereum, would prefer to not change it.

Additionally, web3.js will be replaced and deprecated in the future, making no sense to poke at the typo.